recipes/pdftools/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.37" %}
+{% set sha256 = "634cb50bcd276dda5ab39e3602acfb0370694b61b70eb357201f2bd40240ab4e" %}
+
+package:
+  name: pdftools
```

Comment:
Ignore me, apparently I didn't aid there at all. I vote to stick with pypi - if someone has a non pypi package that they are maintaining then they are sort of 'outside' the Python packaging ecosystem. In this case I feel like the best we can do is raise this upstream and wait?